crates/firewheel-graph/src/graph/compiler/schedule.rs

```diff
@@ -544,6 +544,7 @@ fn sum_inputs(
 }
 
 #[inline]
+#[allow(clippy::mut_from_ref)]
```

Comment:
Clippy will throw an error here without this attribute, so I felt it would be a helpful addition for those of use who use it.